### ISSUE TITLE
Can not create AHT instance with docker-compose 

### ISSUE THREAD

**Issue by holgerbrandl** [eclipse-arrowhead/core-java-spring#271]
When following the description under `docker/README.md` to setup an AHT system, it fails when fetching containers with:
```
brandl@ubuntu2:docker$ docker-compose up --build
Pulling certificateauthority (svetlint/certificateauthority:latest)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling certificateauthority (svetlint/certificateauthority:latest)...
ERROR: pull access denied for svetlint/certificateauthority, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
Also when negating the question _Continue with the new image?_ the problem remains.

I've double-checked https://hub.docker.com/u/svetlint, and indeed there does not seem to be any public container named `svetlint/certificateauthority`

**Comment by tsvetlin** [comment]
Hello!

That image is missing from the docker image repository

**Comment by holgerbrandl** [comment]
I figured that already as described in the initial description. Imho it's a bug either way (docs or implementation)

**Comment by tsvetlin** [comment]
@holgerbrandl The image has been added. I'll update the docker-compose file with the other core system information also. I'll retest everything to make sure everything works.

**Comment by holgerbrandl** [comment]
thanks @tsvetlin .  We'll have a look as soon as the new images are on dockerhub. FYI @awosys

**Comment by wentasah** [comment]
This issue still exists, because the `docker-compose.yml` refers to `svetlint/certificateauthority:latest`, whereas the dockerhub has `svetlint/certificate-authority:latest` (with an extra dash).

**Comment by tsvetlin** [comment]
Hello!

This issue was fixed in Arrowhead 4.4.0 (currently under code review by Eclipse).
Latest (4.4.0) code can be found in the development branch
https://github.com/eclipse-arrowhead/core-java-spring/blob/development/docker/

Best wishes,
Svetlin

**Comment by borditamas** [comment]
Hello,

we have just released the v4.6.1, where a more flexible docker approach has been taken and bug fixes were managed.
[Take a look](https://github.com/eclipse-arrowhead/core-java-spring/wiki/Docker) and please open a new issue if you still runs into some error.
